bin/add-extension-package.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script to add a package to `packages` or `extensions`
+ */
+'use strict';
+
+const build = require('../packages/build');
+const path = require('path');
+const cwd = process.cwd();
+
+let name = process.argv[2];
+if (name == null) {
+  console.error(
+    'Usage: %s <package-name>',
+    path.relative(cwd, process.argv[1]),
+  );
+  process.exit(1);
+}
+
+if (name.startsWith('@loopback/')) {
+  name = name.substring('@loopback/'.length);
+}
+
+const rootPath = path.resolve(__dirname, '..');
+let location = path.relative(rootPath, cwd);
+if (location !== 'packages' && location !== 'extensions') {
+  location = 'extensions';
```

Comment:
Please print a message to tell the user about the location picked by the tool.

We may want to ask the user to confirm the final location before proceeeding to create the new project.

We also may want to ask few questions about the new package, e.g. what `description` to put into `package.json`.

Reply:
By default, the interactive prompts will come up unless `--yes` or `-y` is specified.